pepys_admin/merge.py

```diff
@@ -198,7 +198,6 @@ def update_master_from_slave_entry(self, master_entry, slave_entry):
                         table=master_entry.__table__.name,
                         row_id=getattr(master_entry, primary_key),
                         field=col_name,
-                        new_value=getattr(slave_entry, col_name),
```

Comment:
@BarisSari - shouldn't this be replaced with `previous_value`?

Reply:
No, because we're assigning value of `slave_entry's` `col_name` field to `master_entry's` `col_name` field. So, `master_entry` has a new value. And we don't need to store its previous value, because it's `None`:

https://github.com/debrief/pepys-import/blob/60ea13d8092612cb6ede79682f9fd15b97c34fee/pepys_admin/merge.py#L191-L202

Reply:
Aah yes, of course :-)

I should have expanded the code a little more.